src/test/regress/expected/multi_cross_shard.out

```diff
@@ -0,0 +1,51 @@
+--
+-- MULTI_CROSS_SHARD
+--
+-- Tests to log cross shard queries according to error log level
+--
+ALTER SEQUENCE pg_catalog.pg_dist_shardid_seq RESTART 1440000;
+ALTER SEQUENCE pg_catalog.pg_dist_jobid_seq RESTART 1440000;
+-- Create a distributed table and add data to it
+CREATE TABLE t1(id int, name varchar(20));
+SELECT create_distributed_table('t1', 'id');
+ create_distributed_table 
+--------------------------
+ 
+(1 row)
+
+INSERT INTO t1 VALUES(1, 'elem_1');
+INSERT INTO t1 VALUES(2, 'elem_2');
+INSERT INTO t1 VALUES(3, 'elem_3');
+-- Shouldn't log anything when the log level is 'off'
+SHOW citus.log_dist_statement;
+ citus.log_dist_statement 
+--------------------------
+ off
+(1 row)
+
+SELECT * FROM t1;
+ id |  name  
+----+--------
+  1 | elem_1
+  3 | elem_3
+  2 | elem_2
+(3 rows)
+
+-- Get warning messages with the log level 'warning'
+SET citus.log_dist_statement TO warning;
```

Comment:
Default error level is `notice`, you don't need to set it to `warning`.
https://www.postgresql.org/docs/9.6/static/runtime-config-logging.html#RUNTIME-CONFIG-SEVERITY-LEVELS